<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServerDialog</class>
 <widget class="QDialog" name="TcpServerDialog">
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item><widget class="QLineEdit" name="lineEditPort"><property name="placeholderText"><string>Port</string></property></widget></item>
     <item><widget class="QPushButton" name="startButton"><property name="text"><string>Start</string></property></widget></item>
     <item><widget class="QPushButton" name="stopButton"><property name="text"><string>Stop</string></property></widget></item>
    </layout>
   </item>
   <item><widget class="QTextEdit" name="textEditIncoming"/></item>
   <item>
    <layout class="QHBoxLayout">
     <item><widget class="QLineEdit" name="lineEditMessage"><property name="placeholderText"><string>Message</string></property></widget></item>
     <item><widget class="QPushButton" name="sendButton"><property name="text"><string>Send</string></property></widget></item>
     <item><widget class="QPushButton" name="clearButton"><property name="text"><string>Clear</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
